### ISSUE TITLE
Add short versions of tower tooltips

### ISSUE THREAD

**Issue by Kvel2D** [Praytic/youtd2#241]
Short:
![2023-10-14 17_35_13-Warcraft III](https://github.com/Praytic/youtd2/assets/13269470/8ccfdfb8-590d-46d4-85ad-272b48cd404a)
Long:
![2023-10-14 17_35_28-Warcraft III](https://github.com/Praytic/youtd2/assets/13269470/7f1ca42b-fbb0-400c-bedd-53a3a8060248)

Short version is shown for build button and upgrade button. Short version should display minimal info about abilities.

Short version should show info about stat-based specials, bounce attack, splash attack and multishot:
![2023-10-14 17_39_35-Warcraft III](https://github.com/Praytic/youtd2/assets/13269470/3e2a962a-406d-450e-ac50-8f0bbab335a9)

Short version should show cd, range and cost of autocasts:
![2023-10-14 17_39_22-Warcraft III](https://github.com/Praytic/youtd2/assets/13269470/efa4a121-b452-44b2-be9e-1adb8392b3a0)

Long version is displayed in tower menu. Tower menu has a scroll container so length of tooltips is not a problem there.
